backend/src/main/java/com/shipping/backend/entities/PackageTypeResponse.java

```diff
@@ -0,0 +1,40 @@
+package com.shipping.backend.entities;
+
+import java.io.Serializable;
+
+public class PackageTypeResponse implements Serializable {
```

Comment:
Why do you have `Response` on this class if this is an entity (POJO) by itself?

Reply:
I try to be more explicit with the namig, but in fact I have some troubles with naming convention as you mention on the first comment, so Im already working on that.